editor/plugins/script_editor_plugin.cpp

```diff
@@ -3841,6 +3838,11 @@ void ScriptEditor::_set_zoom_factor(float p_zoom_factor) {
 	}
 }
 
+void ScriptEditor::_on_close_button_clicked() {
```

Comment:
```suggestion
void ScriptEditor::_on_find_in_files_close_button_clicked() {
```
Makes it clear it's not a ScriptEditor's button